src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+  @Override
+  public void doWork() {
+    System.out.println("Bulldozer started its work");
```

Comment:
The message should indicate that it is a `Bulldozer` that is working. Consider including the type of machine in the message to make it more informative as per the checklist.